temporal-sdk/src/main/java/io/temporal/internal/activity/ActivityTaskExecutors.java

```diff
@@ -117,15 +117,23 @@ public ActivityTaskHandler.Result execute(ActivityInfoInternal info, Scope metri
         boolean local = info.isLocal();
         if (ex instanceof ActivityCanceledException) {
           log.info(
-              "{} canceled. ActivityId={}, activityType={}, attempt={}",
+              "{} canceled. Namespace={}, TaskQueue={}, WorkflowId={}, RunId={}, ActivityId={}, activityType={}, attempt={}",
```

Comment:
All of this Data is already on the MDC context so it can be included in any log from the activity woker

Reply:
https://github.com/temporalio/sdk-java/blob/6c4c1835f261e3a6539292809976fdaea9d3fbaa/temporal-sdk/src/main/java/io/temporal/internal/worker/ActivityWorker.java#L245